Makefile

```diff
@@ -28,26 +28,14 @@ endif
 RPM_BUILD_OPTIONS := --with=mysql
 
 OSUSE_REPOS = https://download.opensuse.org/repositories
-ifneq ($(REPOSITORY_URL),"")
-ifneq ($(DAOS_STACK_SLES_12_GROUP_REPO),"")
-sle12_REPOS += --repo $(REPOSITORY_URL)$(DAOS_STACK_SLES_12_GROUP_REPO)
-endif
-ifneq ($(DAOS_STACK_SLES_12_LOCAL_REPO),"")
-sle12_REPOS += --repo $(REPOSITORY_URL)$(DAOS_STACK_SLES_12_LOCAL_REPO)
-endif
-ifneq ($(DAOS_STACK_LEAP_42_GROUP_REPO),"")
-sl42_REPOS += --repo $(REPOSITORY_URL)$(DAOS_STACK_LEAP_42_GROUP_REPO)
-endif
-ifneq ($(DAOS_STACK_LEAP_42_LOCAL_REPO),"")
-sl42_REPOS += --repo $(REPOSITORY_URL)$(DAOS_STACK_LEAP_42_LOCAL_REPO)
-endif
-else
+ifeq ($(REPOSITORY_URL),)
 ifneq ($(ID),centos)
 ADD_REPOS = "munge"
```

Comment:
Curious why only EL7 needs to have the munge repo added.

Wouldn't it make sense to get munge up to date first so that it publishes to Nexus and then remove it from `ADD_REPOS` in this patch?

Reply:
EL7 does not need the munge repo.
Only SLES-12.3/LEAP-42.3 need the munge repo and that is already in Nexus.

This code is to allow building with out using a repository like Nexus.

Reply:
So you don't want the `ADD_REPOS` here then do you?  In any case, it's name was changed to `PR_REPOS`.